pombola/core/static/sass/_common.scss

```diff
@@ -29,19 +29,79 @@ ul.ui-autocomplete {
     float: right;
 }
 
+.grid-listing {
+    overflow: auto;
+
+    @include display-flex();
+    @include flex-wrap(wrap);
+
+    @media (min-width: 40em){
+        margin: -10px;
+    }
+
+    * {
+        box-sizing: border-box;
+    }
+}
+
 .grid-listing-item {
+    box-sizing: border-box;
     float: left;
-    width: 210px;
-    margin: 10px;
-    padding: 5px;
-    overflow: hidden;
+    width: 100%;
+    padding: 10px;
+
+    @media (min-width: 20em){
+        width: 50%;
+    }
+
+    @media (min-width: 30em){
+        width: 33%;
+    }
+
+    @media (min-width: 45em){
+        width: 25%;
+    }
 
-    span.name {
+    a {
+        display: block;
+        padding: 10px;
+        @include box-shadow(rgba(0, 0, 0, 0.2) 0 1px 2px);
+        background-color: #f2f0e7;
+        height: 100%;
+
+        @media (min-width: 40em){
+            padding: 1em;
+        }
+
+        &:hover,
+        &:focus {
+            background-color: darken(#f2f0e7, 5%);
+
+            .parent-place {
+                color: inherit;
+            }
+        }
+    }
+
+    img {
+        max-width: 100%;
+        height: auto;
+    }
+
+    .name {
         font-size: 120%;
         display: block;
-        height: 2.5em;
-        overflow: hidden;
+        margin-top: 0.5em;
+    }
+
+    .place {
+        display: block;
+        margin-top: 0.5em;
     }
 
-    @include box-shadow(rgba(0, 0, 0, 0.3) 1px 1px 12px);
+    .parent-place {
+        display: block;
+        color: #a7a28d;
+        margin-top: 0.1em;
+    }
```

Comment:
I think the changes in this file need to be made to one that's specific to Kenya (somewhere in `pombola/kenya/static/sass/`) rather in this file, which is shared between countries.

As a bonus that should fix the Travis build, which is currently broken because `pombola/core/static/sass/admin.scss` includes `pombola/core/static/sass/_common.scss` but not `pombola/core/static/sass/_core.scss` so it's complaining about `Undefined mixin 'display-flex'` when trying to build `admin.scss`.